src/main/java/com/cupid/jikting/common/jwt/filter/JwtAuthenticationProcessingFilter.java

```diff
@@ -0,0 +1,89 @@
+package com.cupid.jikting.common.jwt.filter;
+
+import com.cupid.jikting.common.error.ApplicationError;
+import com.cupid.jikting.common.error.InvalidJwtException;
+import com.cupid.jikting.common.jwt.service.JwtService;
+import com.cupid.jikting.member.entity.Member;
+import com.cupid.jikting.member.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.mapping.GrantedAuthoritiesMapper;
+import org.springframework.security.core.authority.mapping.NullAuthoritiesMapper;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@RequiredArgsConstructor
+@Slf4j
+public class JwtAuthenticationProcessingFilter extends OncePerRequestFilter {
+
+    private static final String LOGIN_URI = "/members/login";
+
+    private final JwtService jwtService;
+    private final MemberRepository memberRepository;
+    private final GrantedAuthoritiesMapper authoritiesMapper = new NullAuthoritiesMapper();
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response,
+                                    FilterChain filterChain) throws ServletException, IOException {
```

Comment:
p2. throws 앞에서 개행하는건 어떨까요?

```suggestion
    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
            throws ServletException, IOException {
```